api/docs/release.dox

```diff
@@ -145,6 +145,13 @@ compatibility changes:
  - Added the workload to the top bits of the tid for multi-workload drmemtrace
    scheduling.  The drmemtrace view tool now prints the workload and tid in
    the form "Wn.Tmmm".
+ - Added a new directory: `aux` to the scheduler's ignore list when going through the
+   files of the trace directory.  This allows to add additional auxiliary files in the
+   trace directory (under `aux`) without increasing DynamoRIO's versions. Since the
```

Comment:
> without increasing DynamoRIO's versions

Instead of the above, better to say in this sentence: without adding an exception for each new file.